examples/gno.land/r/faucet/faucet.gno

```diff
@@ -9,60 +9,88 @@ import (
 
 var (
 	// configurable by admin.
-	gAdminAddr       std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gControllerAddr  std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gPerTransferSend             = std.Coins{std.Coin{"ugnot", 1000000}}
-	gInPause                     = false
-	gMessage                     = "# Community Faucet.\n\n"
+	gAdminAddr   std.Address    = "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5"
+	gControllers [3]std.Address = [3]std.Address{} // limit it to 3
+
+	gInPause = false
+	gMessage = "# Community Faucet.\n\n"
 
 	// internal vars, for stats.
 	gTotalTransferred std.Coins
 	gTotalTransfers   = uint(0)
+
+	// per account request limit 350 gnot
+	limit std.Coins = std.Coins{std.Coin{"ugnot", 350000000}}
 )
 
-func Transfer(to std.Address) error {
+func Transfer(to std.Address, send int64) string {
 	if err := assertIsController(); err != nil {
-		return err
+		return err.Error()
 	}
 
 	if gInPause {
-		return errors.New("faucet in pause")
+		return errors.New("faucet in pause").Error()
 	}
 
-	send := gPerTransferSend
+	var sendCoins std.Coins
+
+	if send <= limit.AmountOf("ugnot") {
+
```

Comment:
```suggestion
```

your PRs are often hard to review. Could you configure your editor to use automatically a linter, or maybe run `make fmt` before pushing your code, please?

Reply:
Sure. I will use make fmt.